packages/playground/src/components/execution/execution-status.tsx

```diff
@@ -27,16 +27,16 @@ const Badge: React.FC<ExecutionStatusProps & BadgeProps> = props => {
     case ExecutionState.RUNNING:
       return minimal ? (
         <div className="flex items-center gap-1">
-          <div className="w-2 h-2 bg-[rgba(226,155,54,1)] rounded-full" />
+          <div className="w-2 h-2 bg-studio-3 rounded-full" />
```

Comment:
`studio-3` is not a good name. 
we should use naming which indicates what it's used for.. eg. `badge` or `status`

Reply:
Yeah I agree, I've found it challenging to come up with good names, we can discuss how to go about it?